experimenter/experimenter/nimbus-ui/src/components/PageSummary/index.tsx

```diff
@@ -261,13 +261,14 @@ const PageSummary = (props: RouteComponentProps) => {
       >
         {status.draft &&
           !experiment.isArchived &&
-          (showLaunchToReview ? (
+          /* !experiment.canPublishToPreview || */ (showLaunchToReview ? (
```

Comment:
Hrmm? What's this here?

Reply:
That was so I could manually test that the correct error message was being returned by the API. Forgot to uncomment this, whoops